src/kernel.h

```diff
@@ -69,4 +69,6 @@ int64_t CalculateStakeWeightV8(
     const CTransaction &CoinTx, unsigned CoinTxN,
     const MiningCPID &BoincData);
 
+unsigned int GetNumberofStakeOutputs(int64_t nValue);
```

Comment:
Definition in miner but declaration in kernel.h? Make it consistent, please.

Reply:
You don't even have a CreateCoinStake declaration in a header file @tomasbrod ... And for that matter, I didn't make a declaration for SplitCoinStakeOutput... so I need to fix that. I don't even know why the compiler is not barking about it, or about your original CreateCoinStake function that has no declaration. I will move the GetNumberofStakeOutputs and also add SplitCoinStakeOutput to miner.h.

Reply:
I do not have declaration, because the function is not used nowhere, but the file it is defined in. Yes, correctly is should have a declaration or be made static. I am commenting on the fact that you put the declaration to kernel.cpp

Reply:
you do not need declaration if the function is used below it's definition (definition is a declaration)

Reply:
It's actually good to leave it out so the user API becomes more clear. It does make testing a bit more difficult as you have to extern declare it (so don't make it static). You could argue that you should test through the public API. While true, in this case, I suspect the test will become too hard to both read and write.

Reply:
Yep. Really hard to unit test this. I moved the declarations to miner.h. Are you guys ok with that?